calico-enterprise_versioned_docs/version-3.17/releases.json

```diff
@@ -1,257 +1,253 @@
-        "version": "master"
+        "version": "v3.17.0"
       },
       "cnx-node": {
         "image": "tigera/cnx-node",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "dikastes": {
         "image": "tigera/dikastes",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "dex": {
         "image": "tigera/dex",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "fluentd": {
         "image": "tigera/fluentd",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "fluentd-windows": {
         "image": "tigera/fluentd-windows",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "es-proxy": {
         "image": "tigera/es-proxy",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "eck-kibana": {
         "version": "7.17.9"
       },
       "kibana": {
         "image": "tigera/kibana",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "eck-elasticsearch": {
         "version": "7.17.9"
       },
       "elasticsearch": {
         "image": "tigera/elasticsearch",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "cloud-controllers": {
         "image": "tigera/cloud-controllers",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "elastic-tsee-installer": {
         "image": "tigera/intrusion-detection-job-installer",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "es-curator": {
         "image": "tigera/es-curator",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "intrusion-detection-controller": {
         "image": "tigera/intrusion-detection-controller",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "compliance-controller": {
         "image": "tigera/compliance-controller",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "compliance-reporter": {
         "image": "tigera/compliance-reporter",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "compliance-snapshotter": {
         "image": "tigera/compliance-snapshotter",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "compliance-server": {
         "image": "tigera/compliance-server",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "compliance-benchmarker": {
         "image": "tigera/compliance-benchmarker",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "ingress-collector": {
         "image": "tigera/ingress-collector",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "l7-collector": {
         "image": "tigera/l7-collector",
-        "version": "master"
+        "version": "v3.17.0"
       },
       "license-agent": {
         "image": "tigera/license-agent",
-        "version": "master"
-      },
-      "linseed": {
```

Comment:
I believe we need `linseed` section for CE v3.17.

Reply:
@asincu Need your inputs.

Reply:
We need the image to be present here.